zcutil/afl/afl-build.sh

```diff
@@ -1,19 +1,100 @@
 #!/usr/bin/env bash
-# A wrapper around ./zcutil/build.sh for instrumenting the build with AFL:
-#   ./zcutil/afl/afl-build.sh <directory where AFL is installed> <fuzz case>
-# You may obtain a copy of AFL using ./zcutil/afl/afl-get.sh.
 
 set -eu -o pipefail
 
-export AFL_INSTALL_DIR=$(realpath "$1")
-FUZZ_CASE="$2"
-shift 2
+AFL_HARDEN=1
+CONFIGURE_FLAGS="--enable-tests=no --enable-fuzz-main"
+ZCUTIL=$(realpath "./zcutil")
 export AFL_LOG_DIR="$(pwd)"
-export ZCUTIL=$(realpath "./zcutil")
+
+FUZZ_OPTIONS_STRING="Options are: CheckBlock, DecodeHexTx, DeserializeAddrMan, DeserializeTx or ReadFeeEstimates"
```

Comment:
So the plan is, once we're happy with performance etc.. to focus on massively expanding this list. I don't think we should put that here, or maybe we could do like `ls ./src/fuzzing` or something like that so we don't have to maintain two lists.

Reply:
Applied for all at https://github.com/zcash/zcash/pull/4488/commits/a17951bfa806f1795e865289d3a46c0c2225bc1f